src/site/content/en/blog/performance-tool-score-diffs/index.md

```diff
@@ -0,0 +1,188 @@
+
+## Terminology
+
+Various terms about speed/performance are referenced in our tools. We want to start off with some clarification of how these terms are used.
+
+- **Page speed** refers to the speed of an individual page on your site (URL level).
+- **Site speed** refers to the speed of all of the pages on your site (origin level).
+
+It is important to note that "performance" or "speed" is not to be defined by a single metric (i.e. [First Contentful Paint (FCP)](/first-contentful-paint/), [Time to Interactive (TTI)](/interactive/), etc.). This is motivated by several considerations, including:
+- Page load is not a single moment in time - it's an experience that no one metric can fully capture. [source](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics)
+- No single metric in isolation is able to give a meaningful indication of the health of the web (critical for benchmarking).
+- Depending on which metric is chosen, a very different picture of page speed is presented (each requiring different optimizations to improve).
+
+## Performance Scores within tools
+
+The performance score (top level gauge, 1-100 score) in PSI and Lighthouse is calculated from a weighted blend of multiple metrics to summarize a website's/web app's speed. This [weighted blend](https://docs.google.com/spreadsheets/d/1sH_T4G_RZAg4CpcV1bT-tmUegBdBpCOOwsdzqtWnO4U/edit#gid=0) of metrics that together constitute the performance score in both PSI and LH (as of May 2019, [LH v5.0](https://github.com/GoogleChrome/lighthouse/releases)) is as follows:
+- 3X - First Contentful Paint (FCP)
+- 1X - First Meaningful Paint (FMP)
+- 2X - First CPU Idle (FCPUI)
+- 5X - Time to Interactive (TTI)
+- 4X - Speed index (SI)
+- 0X - Max Potential First Input Delay (MPFID)
+
+{% Aside %}
+To learn more see the _[Lighthouse variance 1-pager.](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics)_
+{% endAside %}
+
+TMS uses only FCP (from both Lighthouse and from the CrUX) to provide both site and page speed scores.
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Tool Name</th>
+        <th>Lab / Field Tool</th>
+        <th>Lab Metrics*</th>
+        <th>Field Metrics*</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="https://www.thinkwithgoogle.com/intl/en-gb/feature/testmysite">Test My Site</a></td>
+        <td>Lab & Field</td>
+        <td>FCP**</td>
+        <td>FCP**</td>
```

Comment:
Whoah, this table scares me. Have we explained lab vs field? What do all the abbreviations mean? Asterix descriptions also are a bit vague. I completely understand how hard it is to make this table work! But I do think it might need a few gos.